packages/service-library/src/servicelib/utils.py

```diff
@@ -15,14 +15,26 @@
 
 def is_production_environ() -> bool:
     """
-        If True, this code most probably
-        runs in a production container of one of the
-        osparc-simcore services.
+    If True, this code most probably
+    runs in a production container of one of the
+    osparc-simcore services.
     """
     # WARNING: based on a convention that is not constantly verified
     return os.environ.get("SC_BUILD_TARGET") == "production"
 
 
+def get_http_client_request_total_timeout() -> int:
```

Comment:
In order to keep the defaults and environ-names in sync, you might want to add these helpers in 
``models-library/src/models_library/settings/http_clients.py`` instead.

In any case, ``servicelib`` **already has a dependency** to ``models_library``, right?

Reply:
I've looked again into it again. `servicelib ` dose not have a dependency on `models_library`, I was unsure of adding it.